src/pysdmx/model/__base.py

```diff
@@ -269,6 +283,7 @@ class DataflowRef(Struct, frozen=True, omit_defaults=True):
     agency: str
     id: str
     version: str = "1.0"
+    name: Optional[str] = None
```

Comment:
Missing documentation on name attribute, how would it be used?

Reply:
Thanks for spotting this, @javihern98. I have updated documentation accordingly.